### ISSUE TITLE
Mypy does not recognize library stub

### ISSUE THREAD

**Issue by uSpike** [maxfischer2781/asyncstdlib#61]
When trying to run `mypy` (0.910) on files that import `asyncstdlib` I get the following error:
```
(venv) jordan@enigma:/tmp/tmp.P8Y5Y2yOnD$ mypy test.py 
test.py:1: error: Cannot find implementation or library stub for module named "asyncstdlib"
test.py:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 1 source file)
```

My environment:
```
(venv) jordan@enigma:/tmp/tmp.P8Y5Y2yOnD$ cat test.py
import asyncstdlib
(venv) jordan@enigma:/tmp/tmp.P8Y5Y2yOnD$ python --version
Python 3.9.2
(venv) jordan@enigma:/tmp/tmp.P8Y5Y2yOnD$ pip list
Package           Version
----------------- --------
mypy              0.910
mypy-extensions   0.4.3
pip               20.2.3
setuptools        49.2.1
toml              0.10.2
typing-extensions 3.10.0.2
```

**Comment by maxfischer2781** [comment]
In this case it looks as if MyPy is unable to find ``asyncstdlib`` at all. You should ``pip install`` it or [set `MYPYPATH`](https://mypy.readthedocs.io/en/stable/running_mypy.html#unable-to-find-module).

However, it looks like ``asynctsdlib`` does indeed not correctly signal it is typed. Fixing the module discovery makes MyPy still fail because the type hints are not used:
```bash
test.py:1: error: Cannot find implementation or library stub for module named 'asyncstdlib'
test.py:1: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
```

**Comment by uSpike** [comment]
Whoops, indeed, sorry about the obviously wrong environment.

```shell
(venv) jordan@enigma:/tmp/tmp.P8Y5Y2yOnD$ mypy test.py 
test.py:1: error: Skipping analyzing "asyncstdlib": found module but no type hints or library stubs
test.py:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```

```shell
(venv) jordan@enigma:/tmp/tmp.P8Y5Y2yOnD$ pip list
Package           Version
----------------- --------
asyncstdlib       3.10.0
mypy              0.902
mypy-extensions   0.4.3
pip               21.2.4
setuptools        49.2.1
toml              0.10.2
typing-extensions 3.10.0.2
```

**Comment by maxfischer2781** [comment]
Well, you *did* find a bug there. Thanks for making me aware of it.

Looks like I put ``py.typed`` in the wrong location. :/

**Comment by maxfischer2781** [comment]
This should be fixed in the new 3.10.1 version.
